devscripts/lazy_load_template.py

```diff
@@ -0,0 +1,18 @@
+# encoding: utf-8
+# flake8: noqa
```

Comment:
The generated ```youtube_dl/extractors/lazy_extractors.py``` can pass flake8 with some more newlines. How about just ignore this file in ```setup.cfg```?